### PR TITLE
OT-307 #263 Renamed App and added schemes to xcode project

### PR DESCRIPTION
**Pull request by CihanOezkan**
**Link to the given issue**
#263 
https://jira.open-xchange.com/browse/OT-307

**Describe what the problem was / what the new feature is**
Name of App changed
Schemes are missing in XCode Projekt for matching Android flavors

**Describe your solution**
Changed the name of the App
Added schemes to XCode Projekt

### CODE REVIEW

**Comment by Boehrsi** [comment]
Will be review tomorrow, together with @CihanOezkan.

**Comment by Boehrsi** [comment]
Is on hold as some signing / building issues persist.

File: ios/Runner.xcodeproj/xcshareddata/xcschemes/stable.xcscheme [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+         <BuildActionEntry
+            buildForTesting = "YES"
+            buildForRunning = "YES"
+            buildForProfiling = "YES"
+            buildForArchiving = "YES"
+            buildForAnalyzing = "YES">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "97C146ED1CF9000F007C117D"
+               BuildableName = "OX&#x2007;COI&#x2007;Messenger&#x2007;Dev.app"
```
Comment (Boehrsi) [inline ios/Runner.xcodeproj/xcshareddata/xcschemes/stable.xcscheme]: Please check why this line contains .dev and .app (same for the following lines with the same problem).

Comment (CihanOezkan) [reply ios/Runner.xcodeproj/xcshareddata/xcschemes/stable.xcscheme]: Fixed

**Pull request merged**